packetbeat/_meta/kibana/6/dashboard/Packetbeat-overview.json

```diff
@@ -1,1164 +1,1182 @@
+                "field": "@timestamp",
+                "interval": "auto",
+                "min_doc_count": 1
+              },
+              "schema": "segment",
+              "type": "date_histogram"
+            }
+          ],
+          "listeners": {},
+          "params": {
+            "addLegend": false,
+            "addTimeMarker": false,
+            "addTooltip": true,
+            "defaultYExtents": false,
+            "mode": "stacked",
+            "scale": "linear",
+            "setYExtents": false,
+            "shareYAxis": true,
+            "times": [],
+            "yAxis": {}
+          },
+          "title": "Cache transactions",
+          "type": "histogram"
+        }
+      },
+      "id": "Cache-transactions",
+      "type": "visualization",
+      "updated_at": "2019-01-09T20:47:52.257Z",
+      "version": 1
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": []
+          }
+        },
+        "savedSearchId": "d3089370-88cc-11e7-ad9c-db80de0bf8d3",
+        "title": "RPC transactions",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "id": "1",
+              "params": {
+                "field": "type"
+              },
+              "schema": "metric",
+              "type": "cardinality"
+            },
+            {
+              "id": "2",
+              "params": {
+                "customInterval": "2h",
+                "extended_bounds": {},
+                "field": "@timestamp",
+                "interval": "auto",
+                "min_doc_count": 1
+              },
+              "schema": "segment",
+              "type": "date_histogram"
+            }
+          ],
+          "listeners": {},
+          "params": {
+            "addLegend": false,
+            "addTimeMarker": false,
+            "addTooltip": true,
+            "defaultYExtents": false,
+            "mode": "stacked",
+            "scale": "linear",
+            "setYExtents": false,
+            "shareYAxis": true,
+            "times": [],
+            "yAxis": {}
+          },
+          "title": "RPC transactions",
+          "type": "histogram"
+        }
+      },
+      "id": "RPC-transactions",
+      "type": "visualization",
+      "updated_at": "2019-01-09T20:47:52.257Z",
+      "version": 1
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": []
+          }
+        },
+        "savedSearchId": "Packetbeat-Search",
+        "title": "Response times percentiles",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [
+            {
+              "enabled": true,
+              "id": "1",
+              "params": {
+                "field": "responsetime",
```

Comment:
shouldn't this be event.duration?

Reply:
It should. Despite the huge diff I only changed the "Web Transactions" visualization of the overview dashboard. I will cycle back towards the end of this effort and do all of the general parts of the overview dashboard.

Reply:
I would hope this PR does it automatically for you: https://github.com/elastic/beats/pull/9998